roles/keystone/tasks/openidc.yml

```diff
@@ -26,3 +26,45 @@
 - name: enable apache mod auth_openidc
   apache2_module: name=auth_openidc
   notify: reload apache
+
+- name: create metadata folder for multiple oidc configuration files
+  file:
+    path: "/etc/apache2/openidc/metadata/"
+    state: directory
+    owner: www-data
+    group: root
+    mode: 0775
```

Comment:
there is no daemon. According to documentation only www-data accesses the directory. I can change the permissions